lib/mission_control/jobs/engine.rb

```diff
@@ -79,7 +77,9 @@ class Engine < ::Rails::Engine
           MissionControl::Jobs::Current.server = application.servers.first
         end
 
-        puts "\n\nType 'jobs_help' to see how to connect to the available job servers to manage jobs\n\n"
+        if config.mission_control.jobs.show_console_help
```

Comment:
I think you should use `MissionControl::Jobs.show_console_help` here, as the initialization process has completed if you're opening a console.

Reply:
Thanks, that works, done and rebased